mathics/doc/common_doc.py

```diff
@@ -1480,29 +1872,39 @@ class DocumentationEntry:
     Mathics core also uses this in getting usage strings (`??`).
     """
 
-    def __init__(
-        self,
-        doc_str: str,
-        title: str,
-        section: [DocSection],
-        doctests_class=DocTests,
-        doctest_class=DocTest,
-        doctext_class=DocText,
-    ):
+    def __init__(self, doc_str: str, title: str, section: Optional[DocSection] = None):
+        self._set_classes()
```

Comment:
Same idea as in Documentation: specific classes must be specified in the subclasses.